projects/canopy/src/lib/forms/input/input-field.component.scss

```diff
@@ -115,4 +115,11 @@
   &:-webkit-autofill {
     -webkit-background-clip: text;
   }
+
+  // This is a bit ugly, but maybe a compromise
+  @for $var from 1 through 20 {
```

Comment:
we probably need to check the default case for when a size isn't added, and also look at the other end of the scale for when it exceeds the maximum in the iteration.
The `+ 2` is also a bit arbitrary at the moment.